test/e2e/integration/004-managing-links.js

```diff
@@ -0,0 +1,59 @@
+describe( 'Managing links', () => {
+	before( () => {
+		cy.newPost();
+	} );
+
+	const fixedIsOn = 'button.is-selected:contains("Fix toolbar to block")';
+	const fixedIsOff = 'button:contains("Fix toolbar to block"):not(".is-selected")';
+
+	const switchToMode = ( setFixed ) => {
```

Comment:
Nice! If we find it useful in other tests, we might want to create a command for this one.

Reply:
May I suggest improving the naming. Examples:
- `switchToolbarMode( 'fixed' )`, `'block'` to mimic user language
- `setFixedToolbar( b: boolean )` to mimic internal structure (`state.preferences.features.fixedToolbar`)